app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -386,6 +394,45 @@ fun TripItem(
           }
         })
   }
+  // confirmation button for leaving a trip
+  if (leaveTrip) {
+    AlertDialog(
+        onDismissRequest = { leaveTrip = false },
+        title = { Text(text = stringResource(R.string.leave_trip_text)) },
+        text = { Text(stringResource(R.string.leave_trip_confirmation, trip.name)) },
+        confirmButton = {
+          TextButton(
+              onClick = {
+                val userId = Firebase.auth.uid.orEmpty()
+                if (trip.participants.contains(userId)) {
+                  val updatedParticipants = trip.participants.filter { it != userId }
+                  val updatedTrip = trip.copy(participants = updatedParticipants)
+                  tripsViewModel.updateTrip(
+                      updatedTrip,
+                      onSuccess = {
+                        Toast.makeText(
+                                context,
+                                context.getString(R.string.trip_left_text),
+                                Toast.LENGTH_SHORT)
+                            .show()
+                      },
+                      onFailure = { error ->
+                        Toast.makeText(
+                                context,
+                                "Failed to leave the trip: ${error.message}",
```

Comment:
This hardcoded string should also be extracted into the strings.xml file.

Reply:
You're right, changed it.